doc/source/operations/dell-firmware-update.rst

```diff
@@ -0,0 +1,161 @@
+
+.. code-block:: bash
+
+   curl -O https://dl.dell.com/FOLDER11468378M/1/DRMInstaller_3.4.5.938.bin -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/110.0'
+   docker cp DRMInstaller_3.4.5.938.bin dell-drm:/root
+   docker exec -it dell-drm bash
+   cd /root
+   chmod +x DRMInstaller_3.4.5.938.bin
+   ./DRMInstaller_3.4.5.938.bin
+
+Now you can run DRM, and download a new repo (customise the argument to
+``--inputplatformlist`` depending on the targeted hardware):
+
+.. code-block:: bash
+
+   /opt/dell/dellrepositorymanager/DRM_Service.sh &
+   drm --create -r=idrac_repo --inputplatformlist=R640,R6525
+   drm --deployment-type=share --location=/dell_firmware -r=idrac_repo
+
+Note: sometimes the create call had to be run multiple times before it worked,
+with errors relating to ``Unknown platform: R6525``. Restarting the service
+might be required.
+
+Now we have the all the files in the Docker volume, we can start Apache to
+expose the repo. Use this Dockerfile to support TLS:
+
+.. code-block:: dockerfile
+
+   FROM httpd:2.4
+
+   RUN sed -i \
+       -e 's/^#\(Include .*httpd-ssl.conf\)/\1/' \
+       -e 's/^#\(LoadModule .*mod_ssl.so\)/\1/' \
+       -e 's/^#\(LoadModule .*mod_socache_shmcb.so\)/\1/' \
+       -e 's/Listen 80/#Listen 80/' \
+       conf/httpd.conf
+
+Build a Docker image:
+
+.. code-block:: bash
+
+   docker build --network host -t httpd:local .
+
+Generate a self-signed cert:
+
+.. code-block:: bash
+
+   openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout apache.key -out apache.crt
+
+Run the container:
+
+.. code-block:: bash
+
+   docker run -d --name dell-drm-web --network host -v dell_firmware:/usr/local/apache2/htdocs/ -v $PWD/apache.crt:/usr/local/apache2/conf/server.crt -v $PWD/apache.key:/usr/local/apache2/conf/server.key docker.io/library/httpd:local
+
+.. note::
+
+   At this point the repository may contain only old version of the firmwares.
+   Run an update once to make sure the latest files are available (see next
+   section).
+
+Updating the Repo
+=================
+
+At a later date we will want to re-baseline to a new version. The repo
+can be updated:
+
+.. code-block:: bash
+
+   docker exec -it dell-drm bash
+   [root@seed /]# drm --update -r=idrac_repo
+   # check that it has iterated to a new version
+   [root@seed /]# drm -li=rep
+
+   Listing Repositories...
+
+
+   Name               Latest version   Size      Last modified date
+   ----               --------------   ----      -------------
+   idrac_repo         1.01             4.82 GB   1/9/24 2:22 P.M
+
+   # share the new version
+   [root@seed /]# drm --deployment-type=share --location=/dell_firmware -r=idrac_repo:1.01
+   [root@seed /]# ls -ltra /dell_firmware | tail -1
+   -rw-r--r--   1 root root 7103842 Jan  9 14:24 idrac_repo_Catalog.xml
+
+Then update the ``dell_drm_repo`` variable in ``drac-firmware-update.yml`` if
+required.
+
+Manually adding and update file
```

Comment:
Is this section an alternative to the ``Updating the Repo`` section, or is this part required after it? 
If it is an alternative to the previous section, it may be worth mentioning it in the title as an "(alternative method)", if it is not an alternative it may be worth specifying in the title what files they are specifically updating to help differentiate between them.